bin/generate_rest.sh

```diff
@@ -33,6 +33,9 @@ swagger generate client -P rest_model_zrok.Principal -f "$zrokSpec" -c rest_clie
 echo "...generating js client"
 openapi -s specs/zrok.yml -o ui/src/api -l js
 
+echo "...generating ts client"
+openapi-generator-cli generate -i specs/zrok.yml -o sdk/node/sdk/src/zrok/api -g typescript-node
+
```

Comment:
Need to make sure we add the appropriate check for this new generator to the beginning of this script, to prevent it from failing when the generator isn't installed. It also looks like 2 of the 3 that are currently there are also missing their `exit 1` clauses, so we should fix that at the same time.

Reply:
fixed